### PR TITLE
heirloom-doctools: new port

### PR DESCRIPTION
**Pull request by mohd-akram**
###### Tested on
macOS 10.14.4 18E226
Xcode 10.2 10E125

###### Verification
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: textproc/heirloom-doctools/Portfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        n-t-roff heirloom-doctools 160308
+categories          textproc
+platforms           darwin
+supported_archs     i386 x86_64
```
Comment (mf2k) [inline textproc/heirloom-doctools/Portfile]: We don't generally set `supported_arches` like that. Is there a reason you are setting it?

File: textproc/heirloom-doctools/Portfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+license             CDDL
```
Comment (mf2k) [inline textproc/heirloom-doctools/Portfile]: It looks like most of the few other ports with this license are version 1. Can you verify the version?

`license              CDDL-1`

Comment (mohd-akram) [reply textproc/heirloom-doctools/Portfile]: The version used is 1.1. I did a quick grep and found both `CDDL` and `CDDL-1` so I went with the former. Tangential, but perhaps we should be using SPDX.

File: textproc/heirloom-doctools/Portfile (hunk 1 above)
Comment (mohd-akram) [reply textproc/heirloom-doctools/Portfile]: I'm not sure what the guidelines on that are tbh but I've always set it.

File: textproc/heirloom-doctools/Portfile (hunk 2 above)
Comment (mf2k) [reply textproc/heirloom-doctools/Portfile]: If the version used is 1.1 then set it to:

`license CDDL-1.1`

File: textproc/heirloom-doctools/Portfile [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        n-t-roff heirloom-doctools 160308
```
Comment (mf2k) [inline textproc/heirloom-doctools/Portfile]: The version should be set to something more meaningful. This looks like a date. So add a version in the ISO format YYYY-MM-DD. In this, I'm guessing it should be:

`version 2016-03-08
`

File: textproc/heirloom-doctools/Portfile (hunk 1 above)
Comment (mf2k) [reply textproc/heirloom-doctools/Portfile]: So basically you are saying that this software is not supported on powerpc? If know that for a fact, then keep this line, if not, then it should be removed.

**Comment by mf2k** [comment]
Thank you!

**Pull request merged**